docs/azure.rst

```diff
@@ -80,8 +80,8 @@ For example::
 		}
 	}
 
-The files available in the File share will be available to the task in the directory:
-<YOUR MOUNT DESTINATION>/<YOUR SOURCE FILE SHARE NAME>.
+The files in the File share are available to the task in the directory:
+`<YOUR MOUNT DESTINATION>/<YOUR SOURCE FILE SHARE NAME>`.
 
```

Comment:
Maybe it should be added a basic example to make it more intuitive.

Reply:
See the lines below :)

Reply:
https://github.com/nextflow-io/nextflow/blob/acb17bbff46ac60f8aebf160b2336a9437ef23f8/docs/azure.rst#L86-L100